django/db/backends/base/schema.py

```diff
@@ -142,6 +142,9 @@ def execute(self, sql, params=()):
                 cursor.execute(sql, params)
 
     def quote_name(self, name):
+        # Check if name is an expression
+        if name.startswith('(') and name.endswith(')'):
```

Comment:
I don't get yet in which cases this occurs / why we need this. Shouldn't `quote_name` only be called for columns and table names and such? Calling it with an expression seems wrong.

Reply:
Yes that does feel wrong. I’ll look into it.